packages/ui-components/src/components/Chart/ChartTooltip.js

```diff
@@ -149,7 +149,7 @@ const NoDataTooltip = ({ payload, periodGranularity }) => {
   );
 };
 
-export const Tooltip = props => {
+export const ChartTooltip = props => {
```

Comment:
We had a couple different `Tooltip`s and three different `Table`s in this package, all being exported at root level, causing errors when importing. I renamed them.